prettier/src/trackUsage.ts

```diff
@@ -0,0 +1,29 @@
+const {
+  findConfig,
+  loadConfig,
+  validateConfig,
+  getAccountId,
+  isTrackingAllowed,
+  getAccountConfig,
+} = require("@hubspot/cli-lib");
+import { resolveConfigFile } from "prettier";
+
+export const trackUsage = () => {
+  const rootPath = resolveConfigFile.sync();
+  if (!rootPath) {
+    return;
+  }
+  console.log("rootpath", rootPath);
+
+  const configPath = findConfig(rootPath);
+  if (!configPath) {
+    return;
+  }
+  console.log("configPath", configPath);
+
+  loadConfig(configPath);
+
+  if (!validateConfig()) {
+    return;
+  }
```

Comment:
TODO: 
- More logic to check `isTrackingAllowed`
- Log message to user to let them know that tracking is enabled and how they can disable it (hubspot config)
- Actually implement tracking. My current thought is that we'd spawn a detached child process to make the actual request